### PR TITLE
Amy/cohort flag changes

### PR DESCRIPTION
**Pull request by missamynicholson**

### CODE REVIEW

**Comment by missamynicholson** [comment]
@tom-parker 
A skeleton example test flag to show new schema:
```
{ 
   "name" : "newFlag",
   "ammitState": true,
   "ammitLiveDate": "2017-02-26T10:44:23.000Z", 
   "ammitSetup" : {
        "isAbTest": true,
        "offset": 0,
        "range": 20,
        "cohorts": [
               "premium"
         ],
         "variants": [
                "v1",
                "v2"
         ]
    }
}
```
1. prefixes changed from `abTest-` to `ammit-` to allow for test and cohort flags.
2. `ammitLiveDate` replaces `abTestCreationDate` and is a timestamp for when the `ammitState` was last set to `true`
3.  cohorts array now sits within the `ammitSetup` object
4.  a new attribute `isAbTest` which is `true` is the flag is a test, and `false` if the flag is a cohort

File: server/models/flag.js [hunk 1]
```diff
@@ -67,12 +66,15 @@ class Flag{
 			safeState : this.safeState,
 			state : this.state,
 			tags : this.tags,
-			cohorts : this.cohorts,
```
Comment (tom-parker) [inline server/models/flag.js]: I think this is fine but I can't say for sure if the top level cohorts is used anywhere else

**Comment by tom-parker** [comment]
👍

**Comment by missamynicholson** [comment]
@tom-parker I won't merge this, given it's 5pm! But it should be ready to go.

**Pull request merged**